docs/source/technical_tutorials/search/searching_the_stac_catalog.ipynb

```diff
@@ -7,7 +7,11 @@
    "source": [
```

Comment:
<p>I'd make this header and "Importing and Installing Packages" bigger by using ##. I think "Searching Collections" and "Searching STAC Items" can stay ### though since they seem to be under "STAC Client"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>